### PR TITLE
Add product import command & add utility to start imports

### PR DESCRIPTION
**Pull request by FinnPaes**
This PR adds the product import command.
This also adds a utility to run the import commands from the Statamic CP.

### CODE REVIEW

File: src/Commands/ImportProducts.php [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+            }
+
+            Rapidez::setStore($siteAttributes['magento_store_id']);
+
+            $products = $productModel::query()
+                ->selectAttributes(['entity_id', 'sku', 'name', 'url_key'])
+                ->lazy();
+            
+            foreach ($products as $product) {
+                static::createEntry(
```
Comment (kevinmeijer97) [inline src/Commands/ImportProducts.php:48]: What happens with the title when making the product content entry? Does it use the product name by default?

Comment (FinnPaes) [reply src/Commands/ImportProducts.php:48]: Yes it does use the product name as title, this happens in the `RapidezStatamicServiceProvider` in the `bootListeners` function.
```php
Event::listen('rapidez-statamic:product-entry-data', fn($product) => [
     'title' => $product->name,
     'slug' => trim($product->url_key),
]);
```

File: src/Commands/ImportProducts.php [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+            $bar->advance();
+        }
+        $bar->finish();
+
+
+        return static::SUCCESS;
+    }
+
+    protected static function createEntry(array $attributes, array $values = [])
```
Comment (indykoning) [inline src/Commands/ImportProducts.php:70]: These functions are the same as the ones for the category right? 
Perhaps it's a good idea to move these to an action to be more DRY

**Review by indykoning — CHANGES_REQUESTED** [review]
Could you update the readme with the changes for the product import 🙂

**Review by royduin — APPROVED** [review]

**Review by indykoning — APPROVED** [review]

**Pull request merged**